Update popup settings section (#2306)
telerik/blazor-docs: _contentTemplates/dropdowns/features.md

| Parameter | Type | Description |
| --- | --- | --- |
| `AnimationDuration` | `int` <br /> (`300`) | Sets the dropdown animation duration in milliseconds. |
| `Class` | `string` | Additional CSS class to customize the appearance of the popup. |

> | `Height` | `string` | The height of the popup. The default value is `"200px"`. If set to `"auto"`, the component will automatically adjust the popup height based on the number of items and available space. Note that if you set the `MaxHeight` parameter, the built-in screen boundary detection will be disabled. This means the component will no longer adjust the dropdown height to fit within the viewport. |

Reviewer: ```suggestion
| `Height` | `string` | The height of the popup. The default value is `"200px"`. If set to `"auto"`, the component will automatically adjust the popup height based on the number of items and available space.<br/>Setting `MaxHeight` disables the built-in screen boundary detection. This means that the component will no longer adjust the dropdown height to fit within the viewport. |
```

| `MinHeight`| `string` | The minimum height of the popup. |
| `MinWidth` | `string` | The minimum width of the popup. |
| `MaxHeight` | `string` | The maximum height of the popup. |
| `MaxWidth` | `string` | The maximum width of the popup. |
| `Width` | `string` | The width of the popup. If you don't specify a value, the dropdown width will match the anchor element width which can help with responsive layouts and 100% widths. |

> The above parameters that modify the popup dimentions, like `Height`, `Width`, `MaxWidth` etc. expect a [valid CSS values]({%slug common-features/dimensions%}). Note that, the `MinHeight` and `MaxHeight` have no effect if the `Height` is always within their range. The min and max values are valuable only when the dropdown height is set to a relative unit or changes at runtime.

Reviewer: ```suggestion
The parameters that modify the popup dimensions (`Height`, `Width`, `MaxWidth`, etc.) expect [valid CSS values]({%slug common-features/dimensions%}). 

The `MinHeight` and `MaxHeight` have no effect if the `Height` is always within their range. The min and max values are useful only when the dropdown height is set to a relative unit or changes at runtime.
```

| Parameter | Type | Description |
| --- | --- | --- |

> | `AnimationDuration` | `int` <br /> (`300`) | Sets the dropdown animation duration in milliseconds. |

Reviewer 2: The description of a parameter should not start with a verb. It should look like a big multi-word noun, except for bool parameters.

```suggestion
| `AnimationDuration` | `int` <br /> (`300`) | The dropdown animation duration in milliseconds. |
```

| `Class` | `string` | Additional CSS class to customize the appearance of the popup. |

> | `Height` | `string` | The height of the popup. The default value is `"200px"`. If set to `"auto"`, the component will automatically adjust the popup height based on the number of items and available space.<br/>Setting `MaxHeight` disables the built-in screen boundary detection. This means that the component will no longer adjust the dropdown height to fit within the viewport. |

Reviewer 2: ```suggestion
| `Height` | `string` <br /> (`"200px"`) | The height of the popup. If set to `"auto"`, the component will automatically adjust the popup height based on the number of items and available space. <br /> Setting `MaxHeight` disables the built-in screen boundary detection. This means that the component will no longer adjust the dropdown height to fit within the viewport. |
```
